core-bundle/src/Resources/contao/elements/ContentGallery.php

```diff
@@ -68,6 +68,9 @@ public function generate()
 			return '';
 		}
 
+		// Make sure we have at least one item per row to prevent division by zero
+		$this->perRow = max((int) $this->perRow, 1);
```

Comment:
```suggestion
		if ($this->perRow < 1)
		{
			$this->perRow = 1;
		}
```

Works without a cast and without having to use `max()`. Probably does not affect the performance but surely reduces the complexity.

Reply:
Or you could use `?: 1` inline. 🙃

Reply:
Does this work with negative numbers?

Reply:
`perRow` is unsigned, so likely never contains a negative number.

Reply:
No, but this is not really a valid input for this field isn't it? (rgxp on 'natural')